### ISSUE TITLE
json_out not working for docker container image

### ISSUE THREAD

**Issue by jayrod** [nabla-c0d3/sslyze#416]
**Describe the bug**
When issuing the json_out parameter the output file is no where to be seen.

**To Reproduce**
1. Run sslyze from using docker run command from documentation
2. results are not stored where you think they should be

**Expected behavior**
When the user adds a json file path for json_out then file should appear locally.. Or have a comand documented to copy it  from the container

**Python environment (please complete the following information):**
 - debian 10
- Python version: 3.7

**Additional context**
Using Docker image

**Comment by J12934** [comment]
This would be a good addition to the docs:

Basically: you can use the `--json_out` parameter when using docker. That the results won't show up directly on your hardrive is to be expected as the container has its own filesystem, different from your host operating system. So the file will be written, you just wont see it.

You can however mount folders to containers. Here's an example how to mount your current folder to the containers `/tmp` folder.

```bash
docker run --rm -v="$(pwd):/tmp/" nablac0d3/sslyze --json_out=/tmp/foo.json --regular example.com
```

**Comment by nabla-c0d3** [comment]
Thanks @J12934 for the clarification. I've updated SSLyze to always print the path to the file where the JSON output was written.

**Comment by nabla-c0d3** [comment]
Released in v3.0.4.
